string leader
string follower
float32 tolerance
---
bool success
